doc/releases/changelog-dev.md

```diff
@@ -111,153 +171,67 @@ Users can specify the control wires as well as the values to control the operati
-      circuit, energy, gradient = opt.step_and_cost(circuit, operator_pool, drain_pool=True)
-      print('Energy:', energy)
-      print(qml.draw(circuit)())
-      print('Largest Gradient:', gradient)
-      print()
-      if gradient < 1e-3:
-          break
-  ```
-  
-   ```pycon
-  Energy: -1.246549938420637
-  0: ─╭BasisState(M0)─╭G²(0.20)─┤ ╭<𝓗>
-  1: ─├BasisState(M0)─├G²(0.20)─┤ ├<𝓗>
-  2: ─├BasisState(M0)─│─────────┤ ├<𝓗>
-  3: ─├BasisState(M0)─│─────────┤ ├<𝓗>
-  4: ─├BasisState(M0)─├G²(0.20)─┤ ├<𝓗>
-  5: ─╰BasisState(M0)─╰G²(0.20)─┤ ╰<𝓗>
-  Largest Gradient: 0.14399872776755085
-
-  Energy: -1.2613740231529604
-  0: ─╭BasisState(M0)─╭G²(0.20)─╭G²(0.19)─┤ ╭<𝓗>
-  1: ─├BasisState(M0)─├G²(0.20)─├G²(0.19)─┤ ├<𝓗>
-  2: ─├BasisState(M0)─│─────────├G²(0.19)─┤ ├<𝓗>
-  3: ─├BasisState(M0)─│─────────╰G²(0.19)─┤ ├<𝓗>
-  4: ─├BasisState(M0)─├G²(0.20)───────────┤ ├<𝓗>
-  5: ─╰BasisState(M0)─╰G²(0.20)───────────┤ ╰<𝓗>
-  Largest Gradient: 0.1349349562423238
-
-  Energy: -1.2743971719780331
-  0: ─╭BasisState(M0)─╭G²(0.20)─╭G²(0.19)──────────┤ ╭<𝓗>
-  1: ─├BasisState(M0)─├G²(0.20)─├G²(0.19)─╭G(0.00)─┤ ├<𝓗>
-  2: ─├BasisState(M0)─│─────────├G²(0.19)─│────────┤ ├<𝓗>
-  3: ─├BasisState(M0)─│─────────╰G²(0.19)─╰G(0.00)─┤ ├<𝓗>
-  4: ─├BasisState(M0)─├G²(0.20)────────────────────┤ ├<𝓗>
-  5: ─╰BasisState(M0)─╰G²(0.20)────────────────────┤ ╰<𝓗>
-  Largest Gradient: 0.00040841755397108586
-  ``` 
+    ```pycon
+    >>> mapped_circuit = qml.map_wires(circuit, wire_map)
+    >>> mapped_circuit()
+    tensor([0.92885434, 0.07114566], requires_grad=True)
+    >>> print(qml.draw(mapped_circuit)())
+    A: ──RX(0.54)─┤  Probs
+    B: ──X────────┤       
+    C: ──Z────────┤       
+    D: ──RY(1.23)─┤   
+    ```
 
 <h3>Improvements</h3>
 
-* Added the `samples_computational_basis` attribute to the `MeasurementProcess` and `QuantumScript` classes to track
-  if computational basis samples are being generated.
+* Added a `samples_computational_basis` attribute to the `MeasurementProcess` and `QuantumScript` classes to track if computational basis samples are being generated.
   [(#3207)](https://github.com/PennyLaneAI/pennylane/pull/3207)
 
-* The parameters of a basis set containing different number of Gaussian functions are easier to 
-  differentiate.
+* The parameters of a basis set containing a different number of Gaussian functions are now easier to differentiate.
   [(#3213)](https://github.com/PennyLaneAI/pennylane/pull/3213)
 
-* Printing `MultiControlledX` now shows the `control_values`.
-[(#3113)](https://github.com/PennyLaneAI/pennylane/pull/3113)
+* Printing a `qml.MultiControlledX` operator now shows the `control_values` keyword argument.
+  [(#3113)](https://github.com/PennyLaneAI/pennylane/pull/3113)
 
-* The matrix passed to `qml.Hermitian` is validated when creating the observable if the input is not abstract.
+* The matrix passed to `qml.Hermitian` is now validated when creating the observable if the input is not abstract.
```

Comment:
A little confused here

Reply:
I think we now explicitly check for hermiticity instead of just accepting any matrix